qemu/tests/sev_dhcert_boot.py

```diff
@@ -0,0 +1,63 @@
+import os
+
+from avocado.utils import process
+from virttest import error_context
+from virttest import env_process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu sev basic test on Milan and above host:
+    1. Check host sev capability
+    2. Generate dhcert and session files
+    3. Boot sev VM with dhcert
+    4. Verify sev enabled
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    error_context.context("Start sev test", test.log.info)
+    timeout = float(params.get("login_timeout", 240))
+
+    try:
+        output = process.system_output(params["sev_enable_check"], shell=True)
+        error_context.context("Host sev capabilities: %s" % output,
+                              test.log.info)
+    except Exception as e:
+        test.cancel("Host sev capability check fail: %s" % e)
+    sev_tool_pkg = params.get("sev_tool_pkg")
+    s, o = process.getstatusoutput("rpm -qa | grep %s" % sev_tool_pkg,
+                                   shell=True)
+    if s != 0:
+        try:
+            process.run("yum -y install %s" % sev_tool_pkg, shell=True)
```

Comment:
FYI:
from virttest import utils_package
install_status = utils_package.package_install(packages)
if not install_status:
    test.cancel("Failed to install {packages}.")

Reply:
Thanks Xueqiang for the suggests, I update patch, could you please help to review?